### DISCUSSION TITLE
GitHub] We had a problem billing your account

### DISCUSSION THREAD

**Discussion by idreamgeek** [community/community#63327] [New to GitHub]
anybody know if there's a live person to contact regarding this billing issue? - already updated my credit card information successfully but apparently they're still getting errors charging my copilot account...

**Comment by queenofcorgis** [comment]
Hi @idreamgeek ! The best place to go for answers to billing questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
